bazel/pocket-ic-tests.bzl

```diff
@@ -92,3 +92,35 @@ pocket_ic_mainnet_test = rule(
     executable = True,
     test = True,
 )
+
+def pocket_ic_test(name, macro, **kwargs):
+    """
+    Runs a test using the pocket-ic-server.
+
+    The test will add //rs/pocket_ic_server:pocket-ic-server as a data dependency
+    and will point the environment variable POCKET_IC_BIN to it.
+
+    Args:
+      name: the name of the test target.
+      macro: which test macro to use like rust_test, rust_test_suite_with_extra_srcs, etc.
```

Comment:
understanding question: I'm not very familiar with Bazel terminology but I thought `rust_test` and the likes were usually called a `rule` ([Rust rules](https://github.com/bazelbuild/rules_rust))? Would it make sense to rename `macro` to `test_rule`? Asking because macro in rust context is already quite overloaded

Reply:
Good point. `test_rule` is indeed a better name. I'll rename. Thanks for the suggestion!